core/src/main/java/org/infinispan/util/logging/events/EventLogSerializer.java

```diff
@@ -0,0 +1,39 @@
+package org.infinispan.util.logging.events;
+
+import java.util.Optional;
+
+import org.infinispan.commons.configuration.io.ConfigurationWriter;
+import org.infinispan.configuration.serializing.ConfigurationSerializer;
+
+/**
+ * @since 14.0
+ */
+public class EventLogSerializer implements ConfigurationSerializer<EventLog> {
+
+   @Override
+   public void serialize(ConfigurationWriter writer, EventLog event) {
+      writer.writeStartElement("log");
+      writer.writeAttribute("instant", event.getWhen().toString());
+      writer.writeAttribute("category", event.getCategory().name());
+
+      writer.writeStartElement("content");
+      writer.writeAttribute("level", event.getLevel().name());
+      writer.writeAttribute("message", event.getMessage());
+      writer.writeAttribute("detail", unwrap(event.getDetail()));
+      writer.writeEndElement();
+
+      writer.writeStartElement("additional");
```

Comment:
IMO this should be called "meta" or "metadata" to make it's purpose more clear.

I also think that the "instant" attribute should be a child of this element, so we have three top level elements of:

```yaml
log: 
  category:
  content:
  meta:
```